include/tango/server/command.h

```diff
@@ -220,9 +222,10 @@ class Command
  * The object desctructor.
  */
 #ifdef HAS_UNIQUE_PTR
-    virtual ~Command() {}
+    virtual ~Command()
+    {}
 #else
-	virtual ~Command() {delete ext;}
+    virtual ~Command() {delete ext;}
```

Comment:
OK, thanks for the tip. Will do it in the master since this PR is already merged